src/main/java/org/zone/Permissions.java

```diff
@@ -9,7 +9,25 @@
     REGION_ADMIN_INFO("zone.region.admin.info"),
     BYPASS_INTERACTION_DOOR("zone.bypass.interaction.door"),
     BYPASS_INTERACTION_BLOCK("zone.bypass.interaction.block"),
-    REGION_EDIT_BOUNDS("zone.region.edit.bounds");
+    REGION_EDIT_BOUNDS("zone.region.edit.bounds"),
+    FLAG_SET("zone.region.flag.set"),
+    FLAG_GET("zone.region.flag.get"),
+    SET_GREETINGS("zone.region.flag.greetings.message.set"),
+    REMOVE_GREETINGS("zone.region.flag.greetings.message.remove"),
+    SET_LEAVING_MESSAGE("zone.region.flag.leaving.message.set"),
+    REMOVE_LEAVING_MESSAGE("zone.region.flag.leaving.message.remove"),
+    OVERRIDE_PLAYER_FALL_DAMAGE("zone.override.fall.damage.player"),
+    OVERRIDE_PLAYER_ENTITY_DAMAGE("zone.override.entity.damage.player"),
+    OVERRIDE_MONSTER_PREVENTION("zone.override.monster.prevention"),
+    OVERRIDE_GREETINGS("zone.override.greetings"),
+    OVERRIDE_LEAVING("zone.override.leaving"),
+    OVERRIDE_PLAYER_PREVENTION("zone.override.prevention.player"),
+    BYPASS_PLAYER_PREVENTION("zone.bypass.prevention.player"),
```

Comment:
Whats the difference between override and bypass?

Reply:
they r same but yes.

Reply:
Remove one then